units/github_status_api.go

```diff
@@ -177,21 +177,17 @@ func (j *githubStatusUpdateJob) preamble() error {
 }
 
 func (j *githubStatusUpdateJob) fetch() (*message.GithubStatus, error) {
-	var patchDoc *patch.Patch
-	status := message.GithubStatus{}
+	status := message.GithubStatus{
+		Owner: j.Owner,
+		Repo:  j.Repo,
+		Ref:   j.Ref,
+	}
 
 	if j.UpdateType == githubUpdateTypeProcessingError {
 		status.Context = j.GithubContext
 		status.State = message.GithubStateFailure
 		status.Description = j.Description
 
-		status.Owner = j.Owner
```

Comment:
What's the reason for removing these?

Reply:
I moved it to the top, since for most of the cases we assign owner/repo/ref from the job params.